spec/lib/license_finder/package_managers/gradle_spec.rb

```diff
@@ -147,46 +148,83 @@ module LicenseFinder
     describe '#active?' do
       include FakeFS::SpecHelpers
 
-      it 'return true if build.gradle exists' do
-        FakeFS do
-          FileUtils.mkdir_p '/fake/path'
-          FileUtils.touch '/fake/path/build.gradle'
+      context 'when dealing with root gradle project' do
+        context "when there's a build.gradle" do
+          it 'returns true' do
+            FakeFS do
```

Comment:
having setup blocks might save some nested code here: 
```
before do
    FakeFS.activate!
  end

  after do
    FakeFS.deactivate!
  end
```

Reply:
I believe this is automatically handled by `include FakeFS::SpecHelpers` which is included in the `#activate?` describe block (line 149)